phpstan.neon

```diff
@@ -37,6 +37,9 @@ parameters:
              message: '#Instead of "instanceof/is_a\(\)" use ReflectionProvider service or "\(new ObjectType\(<desired_type\>\)\)\-\>isSuperTypeOf\(<element_type\>\)" for static reflection to work#'
              path: src/ValueObjectFactory/ServiceMapFactory.php
 
+        - '#Cognitive complexity for "Rector\\Symfony\\Rector\\Closure\\MinimalSharedStringSolver\:\:solve\(\)" is (.*?), keep it under 10#'
```

Comment:
I decided to ignore this error, since I don't want to fiddle in this "external" code and introduce bugs since we don't know all the details